tests/unit/disco/components/TestAddonCompatibilityError.js

```diff
@@ -40,35 +28,29 @@ describe(__filename, () => {
   });
 
   it('renders a notice for iOS users', () => {
-    const root = render({
-      reason: INCOMPATIBLE_FIREFOX_FOR_IOS,
-      userAgentInfo: { browser: { name: 'Firefox' }, os: { name: 'iOS' } },
-    });
+    const root = render({ reason: INCOMPATIBLE_FIREFOX_FOR_IOS });
     const content = root.find('.AddonCompatibilityError').render();
 
     expect(content.text()).toContain(
       'Firefox for iOS does not currently support add-ons.',
     );
   });
 
-  it('renders a generic message when reason code not known', () => {
-    const root = render({ reason: 'fake reason' });
+  it('renders a generic message when not FF', () => {
+    const root = render({ reason: INCOMPATIBLE_NOT_FIREFOX });
```

Comment:
Nit: We don't technically need this test because there is no code path for this reason. This will take the same path as `render({ reason: 'fake reason' })`, but I suppose it's nice to have a test for this as this is a reason that we could expect to actually be set. I'll leave it up to you whether to include this test or remove it.